packages/core/src/App/Containers/ConfirmEmailModal/__tests__/confirm-email-modal.spec.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { mockStore, StoreProvider } from '@deriv/stores';
+import { WS } from 'Services';
+import { ConfirmEmailModal } from '../confirm-email-modal';
+
+jest.mock('@deriv/account', () =>
+    jest.fn(({ onClose, onClickSendEmail }) => (
+        <div>
+            SentEmailModal
+            <button onClick={onClose}>Close</button>
+            <button onClick={onClickSendEmail}>ClickSendEmail</button>
+        </div>
+    ))
+);
+
+jest.mock('Services', () => ({
+    ...jest.requireActual('Services'),
+    WS: {
+        changeEmail: jest.fn(() => Promise.resolve({})),
+    },
+}));
+
+describe('ConfirmEmailModal', () => {
+    let modal_root_el: HTMLElement, mock_props: React.ComponentProps<typeof ConfirmEmailModal>;
+
+    beforeAll(() => {
+        modal_root_el = document.createElement('div');
+        modal_root_el.setAttribute('id', 'modal_root');
+        document.body.appendChild(modal_root_el);
+    });
+
+    afterAll(() => {
+        document.body.removeChild(modal_root_el);
+    });
+
+    beforeEach(() => {
+        jest.clearAllMocks();
+        mock_props = {
+            onClose: jest.fn(),
+            changed_email: 'new_test@example.com',
+            is_open: true,
+            prev_email: 'test@example.com',
+            setErrorMessage: jest.fn(),
+            setEmailValue: jest.fn(),
+        };
+    });
+
+    const modal_text_content = [
+        'Are you sure?',
+        /Are you sure you want to update email/i,
+        'test@example.com',
+        'new_test@example.com',
+    ];
+
+    const mock_store = mockStore({});
+    const wrapper = ({ children }: { children: JSX.Element }) => (
+        <StoreProvider store={mock_store}>{children};</StoreProvider>
+    );
+    const renderComponent = () => {
+        render(<ConfirmEmailModal {...mock_props} />, { wrapper });
+    };
+
+    it('should not render ConfirmEmailModal if is_open is false', () => {
+        mock_props.is_open = false;
+        renderComponent();
+
+        expect(screen.queryByText(/Are you sure?/i)).not.toBeInTheDocument();
```

Comment:
I see that there's 'Are you sure?' and also /Are you sure?/i here, can we use only one of the text? 🤔 then we can just use `modal_text_content[0]` here 😎